www/src/components/Settings.tsx

```diff
@@ -85,9 +90,31 @@ const Settings = () => {
             ))}
           </Form.Control>
         </Form.Group>
+        {!showMinimizedTitle && (
```

Comment:
[curious] Can you clarify this logic? Why don't we want "Container Width" included as an option in this case?

Reply:
I don't think we need to display these options on the main documentation page. What do you think?